### DISCUSSION TITLE
Ordering node in html output

### DISCUSSION THREAD

**Discussion by lukenavo** [freeplane/freeplane#1238] [Help]
Hi, I need some help.
I use freeplane mind map (1.11.4 on Debian 11)  for teaching and normally I need to load them in moodle in xhtml clikable format.
The problem: when I want to show and explain graphical map to my student I used  to start from left upper section (if you can imagine the map in four section) then follow the following ones clockwise (I used a LIM in my classrooms).
This is the way Buzan explain to be good for mapping.
The problem is that nodes in xhtml text part of the map (under the graphical map) doesn't follow this order. Sometimes I arrange node position to meet my requirements but the graphical order (clockwise from upper left section) doesn't meet the text order. 
I try to modify text order using "structure view" but when I change order in this view also change order of the nodes in graphical view. 
Is there a way to get graphical node ordering from text structure node ordering?

Thanks very much for any help

![html graphical order](https://github.com/freeplane/freeplane/assets/63333946/82918e0f-c83d-489d-965b-97f3dcbec434)
![html text order](https://github.com/freeplane/freeplane/assets/63333946/ed98c929-4945-43e7-bbdd-782d03299f61)

**Comment by dpolivaev** [comment]
I don't have any easy solution because Freeplane always orders nodes from the top to the bottom.

**Reply by lukenavo** [reply]
Thank you for the answer. It seems strange that there is no option for this. For example in View your mind (vym) app it is a very symple task: there is a graphic editor and an indipendent tree editor. So the html output is what you decide to be. But there is no comparison between the two app... And, furthermore, it is a load of work convert all my freeplane maps to vym.

**Reply by dpolivaev** [reply]
Don't expect that all apps have the same set of features.
Therefore there are many different apps.
